.travis.yml

```diff
@@ -10,97 +10,96 @@ python:
+  - TOX_ENV="django17"
+  - TOX_ENV="django18"
+  - TOX_ENV="django19"
+  - TOX_ENV="django110"
+  - TOX_ENV="django111"
+  - TOX_ENV="djangodev"
 
 matrix:
+  fast_finish: true
   exclude:
-  - python: "3.6"
-    env: DJANGO="Django>=1.4,<1.5"
-  - python: "3.6"
-    env: DJANGO="Django>=1.5,<1.6"
-  - python: "3.6"
-    env: DJANGO="Django>=1.6,<1.7"
-  - python: "3.6"
-    env: DJANGO="Django>=1.7,<1.8"
-  - python: "3.6"
-    env: DJANGO="Django>=1.8,<1.9"
-  - python: "3.6"
-    env: DJANGO="Django>=1.9,<1.10"
-  - python: "3.6"
-    env: DJANGO="Django>=1.10,<1.11"
+    - python: "3.6"
+      env: TOX_ENV="django14"
+    - python: "3.6"
+      env: TOX_ENV="django15"
+    - python: "3.6"
+      env: TOX_ENV="django16"
+    - python: "3.6"
+      env: TOX_ENV="django17"
+    - python: "3.6"
+      env: TOX_ENV="django18"
+    - python: "3.6"
+      env: TOX_ENV="django19"
+    - python: "3.6"
+      env: TOX_ENV="django110"
 
-  - python: "3.5"
-    env: DJANGO="Django>=1.4,<1.5"
-  - python: "3.5"
-    env: DJANGO="Django>=1.5,<1.6"
-  - python: "3.5"
-    env: DJANGO="Django>=1.6,<1.7"
-  - python: "3.5"
-    env: DJANGO="Django>=1.7,<1.8"
+    - python: "3.5"
+      env: TOX_ENV="django14"
+    - python: "3.5"
+      env: TOX_ENV="django15"
+    - python: "3.5"
+      env: TOX_ENV="django16"
+    - python: "3.5"
+      env: TOX_ENV="django17"
 
-  - python: "3.4"
-    env: DJANGO="Django>=1.4,<1.5"
-  - python: "3.4"
-    env: DJANGO="Django>=1.9,<1.10"
-  - python: "3.4"
-    env: DJANGO="Django>=1.10,<1.11"
-  - python: "3.4"
-    env: DJANGO="Django>=1.11,<1.12"
+    - python: "3.4"
+      env: TOX_ENV="django14"
+    - python: "3.4"
+      env: TOX_ENV="django19"
+    - python: "3.4"
+      env: TOX_ENV="django110"
+    - python: "3.4"
+      env: TOX_ENV="django111"
 
-  - python: "3.3"
-    env: DJANGO="Django>=1.4,<1.5"
-  - python: "3.3"
-    env: DJANGO="Django>=1.9,<1.10"
-  - python: "3.3"
-    env: DJANGO="Django>=1.10,<1.11"
-  - python: "3.3"
-    env: DJANGO="Django>=1.11,<1.12"
-  - python: "3.3"
-    env: DJANGO="https://github.com/django/django/tarball/master"
+    - python: "3.3"
+      env: TOX_ENV="django14"
+    - python: "3.3"
+      env: TOX_ENV="django19"
+    - python: "3.3"
+      env: TOX_ENV="django110"
+    - python: "3.3"
+      env: TOX_ENV="django111"
+    - python: "3.3"
+      env: TOX_ENV="djangodev"
 
-  - python: "3.2"
-    env: DJANGO="Django>=1.4,<1.5"
-  - python: "3.2"
-    env: DJANGO="Django>=1.9,<1.10"
-  - python: "3.2"
-    env: DJANGO="Django>=1.10,<1.11"
-  - python: "3.2"
-    env: DJANGO="Django>=1.11,<1.12"
-  - python: "3.2"
-    env: DJANGO="https://github.com/django/django/tarball/master"
+    - python: "3.2"
+      env: TOX_ENV="django14"
+    - python: "3.2"
+      env: TOX_ENV="django19"
+    - python: "3.2"
+      env: TOX_ENV="django110"
+    - python: "3.2"
+      env: TOX_ENV="django111"
+    - python: "3.2"
+      env: TOX_ENV="djangodev"
+
+    - python: "2.6"
```

Comment:
You're right. I also think it's worth dropping support for old versions of django. That's one for another issue though, as that will have an impact on the main code, and I'd like to limit the scope of this issue only to moving travis to tox.